### DISCUSSION TITLE
Server is not reachable on ios app

### DISCUSSION THREAD

**Discussion by sirjjq** [immich-app/immich#8600] [General]
### The bug

I can type in the ip and port number of my docker container (made using docker compose) into a browser and it will work fine. Trying to login to the ios app using that same ip and port (http://ip:port/api or any combo of this) gives me a server not reachable error. 

### The OS that Immich Server is running on

unraid using docker compose 

### Version of Immich Server

newest version 

### Version of Immich Mobile App

1.101.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
default recommended docker compose
```

### Your .env content

```Shell
default recommended docker compose
```

### Reproduction steps

```bash
1.try to login to ios app
2.fail
3.
...
```

**Comment by stim-ca** [comment]
Just for some clarification, are you able to reach the web gui from your browser on your phone, or like a computer?

I just set up immich yesterday, and am using the ISO app. The server address that I used was http://IP:PORT without the /api. Can you report back if that works.

**Comment by alextran1502** [comment]
Do you allow the app to connect to the local network?

**Comment by sirjjq** [comment]
I don’t see that anywhere in the preferences in the ios login page. Do i have to enable it in the web browser? Thank you!On Apr 7, 2024, at 1:08 PM, Alex ***@***.***> wrote:﻿
Do you allow the app to connect to the local network?

**Reply by stim-ca** [reply]
Open IOS settings scroll down to immich, then press, and verify "Local Network" is toggled.

**Comment by sirjjq** [comment]
That seemed to have fixed it, thanks a ton!On Apr 7, 2024, at 1:31 PM, Shawn Timewell ***@***.***> wrote:﻿
Open IOS settings scroll down to immich, then press, and verify "Local Network" is toggled.

**Comment by qbirdee** [comment]
Having same issue. Reachable via browser, not reachable via app. The problem is that the app never asked for local network access, so my settings do not give me the option to enable local network access.  How do I surface the setting?  Thanks.

**Reply by alextran1502** [reply]
@qbirdee are you on the latest version 1.135.1?

**Reply by domenicoop** [reply]
I have the same exact problem, my version is v1.140.1 for iOS 18.6.2

I cannot toggle the Local Network because it doesn’t appear on the settings menu of iOS.

**Reply by alextran1502** [reply]
@domenicoop Can you try restarting the phone?

**Reply by rharley99** [reply]
i just restarted my phone and im having the same issue there is no prompt for local network! :( i cant get inside the app or on the browser though
